Documentation/building/manually-building.md

```diff
@@ -0,0 +1,64 @@
+# Build and run your .NET Core application with csc and CoreRun
+
```

Comment:
> At the time of writing, required tools like `csc` are not accessible on Unix/MacOS machines

I wouldn't want folks to get the impression that compilation requires Windows. Perhaps just state the instructions are currently written for Windows machines.

Reply:
Also, the csc `is` available on Unix/MacOS (it is downloaded as part of buildtools during the build). I actually tend to use the csc from buildtools myself even on Windows to get the latest compiler.

Reply:
@jkotas how do you invoke csc?

Reply:
`jkotas@jkotas7:~/corefx$ Tools/dotnetcli/dotnet Tools/csc.dll`

Reply:
I do the same.  I also invoke other tools under Tools in a similar fashion, e.g. MSBuild.dll, when running tests in a specific test suite.

Reply:
Oh my. I'm currently using corerun to invoke it which could result in a contract mismatch. I'll change it to the dotnet under dotnetcli. Thanks! I published the docs and already got some good feedback on it.

Reply:
@jkotas your command results for me on Windows in this following error:

```
Could not load file or assembly 'System.Text.Encoding.CodePages, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
```

I noticed @eerhardt was facing the same error. Not sure if that's expected.

Reply:
Yes, the .NET Core-based csc is not wired-in correctly on Windows. It would be nice to fix it... .

I run the desktop version of the tools on Windows, e.g. Tools\net46\roslyn\tools\csc.exe . It is what the build.cmd on Windows does, so it is what works.